solver/pb/caps.go

```diff
@@ -232,6 +232,10 @@ func init() {
 		ID:      CapFileBase,
 		Enabled: true,
 		Status:  apicaps.CapStatusPrerelease,
+		SupportedHint: map[string]string{
+			"docker":   "Docker v19.03",
```

Comment:
Can we consider using Docker REST API version?

Reply:
and wondering hist string should be machine-readable format (JSON)

Reply:
This text only appears inside the error message shown to the user. I don't think Docker user would know about the API versions.

Reply:
For detecting if cap is supported to avoid errors, client should just check the cap directly and no record specific to the product.